From 151468e59714d2346dc1ddff1078d7482f008c1c Mon Sep 17 00:00:00 2001
From: Bernd Waibel <waebbl@gmail.com>
Date: Sat, 22 Feb 2020 17:11:02 +0100
Subject: [PATCH] Patch search for coin

- Check for Coin_FOUND instead of COIN3D_FOUND
- Add aliases for COIN3D_INCLUDE_DIRS and COIN3D_LIBRARIES

Signed-off-by: Bernd Waibel <waebbl@gmail.com>
---
 cMake/FreeCAD_Helpers/SetupCoin3D.cmake | 9 ++++++---
 1 file changed, 6 insertions(+), 3 deletions(-)

diff --git a/cMake/FreeCAD_Helpers/SetupCoin3D.cmake b/cMake/FreeCAD_Helpers/SetupCoin3D.cmake
index b50db9a93..86cec5ccb 100644
--- a/cMake/FreeCAD_Helpers/SetupCoin3D.cmake
+++ b/cMake/FreeCAD_Helpers/SetupCoin3D.cmake
@@ -1,11 +1,14 @@
 macro(SetupCoin3D)
 # -------------------------------- Coin3D --------------------------------
 
-    find_package(Coin3D REQUIRED)
-    if(NOT COIN3D_FOUND)
+    find_package(Coin REQUIRED)
+    if(NOT Coin_FOUND)
         message(FATAL_ERROR "=================\n"
                             "Coin3D not found.\n"
                             "=================\n")
-    endif(NOT COIN3D_FOUND)
+    endif(NOT Coin_FOUND)
 
+    get_property(Coin_INTERFACE_INCLUDE_DIRS TARGET Coin::Coin PROPERTY INTERFACE_INCLUDE_DIRECTORIES)
+    set(COIN3D_INCLUDE_DIRS ${Coin_INTERFACE_INCLUDE_DIRS})
+    set(COIN3D_LIBRARIES ${Coin_LIBRARIES})
 endmacro(SetupCoin3D)
-- 
2.25.0

